Formulaic TOS

```diff
@@ -0,0 +1,73 @@
+#Terms of Service - Mozilla Formulaic
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" 
+
+Formulaic (the “Service”) is a service that allows users to create, share, manage, and use templates (the “Formulas”) for querying Artificial Intelligence (AI) based Large Language Model (LLMs) (the “Service”).  It also allows users to run those queries through a third party LLM platform, and to view the resulting responses. (For the purposes of this Agreement, Formulas, instructions, descriptions, and responses will all be referred to as the “Content”.) The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
+Please read these terms of service (“Agreement” or “Terms”) carefully before you sign up for or use the Service.  The Terms constitute a binding legal agreement, which govern your use of the Service via any platform or device.  By accessing, signing up for, or using the Service, you agree to be bound by these Terms.  
+
+##1. The Service
```

Comment:
Missing space after the dash, missing empty line after a title (across the document).